platform/ethereum/opensea/model.go

```diff
@@ -43,6 +47,10 @@ type Collectible struct {
 	Permalink       string                 `json:"permalink"`
 	Description     string                 `json:"description"`
 	Collection      CollectibleCollections `json:"collection"`
+
+	// only available for ERC1155
+	FeeToken    *TransferFeeToken `json:"transfer_fee_payment_token,omitempty"`
+	TransferFee string            `json:"transfer_fee,omitempty"`
```

Comment:
only `TransferFeeToken` is needed right? 

`TransferFee string` is not needed

Reply:
`TransferFee` is needed too, two fields in opensea response